jib-gradle-plugin/scripts/update_gcs_latest.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -
+# Usage: ./scripts/update_gcs_latest.sh <release version>
+
+set -e
+
+EchoRed() {
+	echo "$(tput setaf 1; tput bold)$1$(tput sgr0)"
+}
+EchoGreen() {
+	echo "$(tput setaf 2; tput bold)$1$(tput sgr0)"
+}
+
+Die() {
+	EchoRed "$1"
+	exit 1
+}
+
+# Usage: CheckVersion <version>
+CheckVersion() {
+    [[ $1 =~ ^[0-9]+\.[0-9]+\.[0-9]+(-[0-9A-Za-z]+)?$ ]] || Die "Version: $1 not in ###.###.###[-XXX] format."
+}
+
+[ $# -ne 1 ] && Die "Usage: ./scripts/update_gcs_latest.sh <release version>"
+
+CheckVersion $1
+
+versionString="{\"latest\":\"$1\"}"
+destination="gs://jib-versions/jib-gradle"
+
+echo $versionString > jib-gradle
+gsutil rm $destination
```

Comment:
I believe this is unnecessary and the file will be overwritten. This is safer in the case `gsutil cp` fails.